### DISCUSSION TITLE
Error de prestashop 8.2.0

### DISCUSSION THREAD

**Discussion by ferrecesar** [PrestaShop/PrestaShop#37343] [Q&A]
Hola buenas, soy nuevo por aqui. Resulta y haber si me puedo explicar al pasar de prestashop 1.7 al 8.2.0 ahora estoy teniendo una serie de problemas y doy pie con bolo, busco ayuda urgente por favor. pongo el error que me da la url tambien, y el modulop de ps_acunts etc...
* https://almacendeferreterialpormayor.com/
*  error en el modulo de fuentes de trafico.
* modulos en depecrate y no soy capad de actualizar
* version de php 8.1
* prestashop 8.2.0
* modulo ps_acunts
* Unknown error en línea 274 en archivo /home/almacendeferreterialpormayor/www/modules/ps_accounts/vendor/monolog/monolog/src/Monolog/Logger.php
[8192] DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated

muchas gracias antemano

**Comment by alisamie97** [comment]
Hi @ferrecesar 

Change php version on your server to 8.0 and don't use any higher version
Seems like PrestaShop has not been updated to be fully compatible with higher PHP versions
